### DISCUSSION TITLE
Debug logs are no longer generated in api/logs

### DISCUSSION THREAD

**Discussion by GauthierPLM** [danny-avila/LibreChat#1784] [Troubleshooting]
### What happened?

Since version v0.6.6, it seems that DEBUG logs are no longer written in api/logs.
I enabled the DEBUG_LOGGING in .env, but no log files are generated.

### Steps to Reproduce

1. Git clone the project.
2. Configure Azure OpenAI.
3. Ensure DEBUG_LOGGING is set to true
4. Start LibreChat via docker compose up -d
5. Check the log files.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If you're using docker, they should be within the container in `app/api/logs`

**Reply by danny-avila** [reply]
just checked, they are indeed in `app/api/logs` as expected within docker. if you expect some where else you would have to do some volume mapping with the override file

![image](https://github.com/danny-avila/LibreChat/assets/110412045/9042a1eb-2015-4cd8-9ae7-36248a4a6887)

https://docs.librechat.ai/install/configuration/docker_override.html

**Reply by GauthierPLM** [reply]
Thank you, I think I understand what is going on:
- [This commit](https://github.com/danny-avila/LibreChat/commit/5d7869d3d52e9e83adebb4d322f9d65af0e9c402#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3L30) changed the Dockerfile and removed the mapping `./api:/app/api`, hence why logs are not present were documented.
- The docker compose does not map the log directory (ex: `./logs:/app/api/logs`.
- The documentation still mention to look into `./api/logs`.

I was able to access the logs by adding the mapping to the docker-compose.override.yaml, but I think it's one of the mappings which should be available by default as it's quite common to try to read logs / debug what's going on, and there is no documentation of this at the moment.

**Reply by danny-avila** [reply]
You're right it should be mounted by default. I'll add that in soon.
